### DISCUSSION TITLE
Where this setting "--int-removal 0" comes from?

### DISCUSSION THREAD

**Discussion by Masonwei32** [vdemichev/DiaNN#1045] [Q&A]
Hello, everyone! I found a setting "--int-removal 0" from the log file, but when I reanalysed the raw data again, I couldn't find this option in the GUI. So I would like to ask whether it was from the software update or just from my mistake to check any option. It would be better if you could also explain what this setting does. I appreciate any comment. Thank you so much!
[DIA.pdf](https://github.com/user-attachments/files/15808465/DIA.pdf)

**Comment by vdemichev** [comment]
This setting no longer applies to DIA-NN, it’s the old high-accuracy quantification mode

**Reply by Masonwei32** [reply]
I see, no wander I couldn't find it in the recent analysis. Thanks for your instant reply!

**Reply by sooheon** [reply]
I have a log with `--no-ifs-removal` and `--int-removal 0`, what settings in GUI do I need to replicate this?

**Reply by vdemichev** [reply]
`--no-ifs-removal` - always the case with 1.9 and later
`--int-removal 0` - uncheck 'no shared spectra' - this is strongly not recommended

**Reply by sooheon** [reply]
This was done on 1.8.1, and I want to recreate precisely, how can I enable `no-ifs-removal` on 1.8.1?

**Reply by vdemichev** [reply]
Yes, high-precision quantification mode

**Reply by vdemichev** [reply]
Btw, you can also just put the options in a config file (referenced with --cfg) or 'Additional options'

**Reply by sooheon** [reply]
Thank you!
